cmd/hypper/install.go

```diff
@@ -65,7 +65,7 @@ func newInstallCmd(actionConfig *action.Configuration, logger log.Logger) *cobra
 			if err != nil {
 				return err
 			}
-			logger.Info(eyecandy.ESPrint(settings.NoEmojis, ":clapping_hands: Done!"))
+			logger.Info(eyecandy.ESPrint(settings.NoEmojis, ":clapping_hands:Done!"))
```

Comment:
Why did you remove the space? Looks better with....

Reply:
it's unclear from the code, but with this change, it looks aligned in my terminal:

![tty](https://user-images.githubusercontent.com/2196685/116872300-ed45d700-ac15-11eb-85d1-7073c05deae3.png)

Curiously, it looks also aligned on emacs:

![emacs](https://user-images.githubusercontent.com/2196685/116872328-fa62c600-ac15-11eb-844a-2ac1dff21149.png)

But it doesn't look aligned in vscode:

![vscode](https://user-images.githubusercontent.com/2196685/116872354-03ec2e00-ac16-11eb-9ab1-cb2ac8fd56ad.png)

nor if I copy-paste the testdata/output and format it here:

```
🛳  Installing shared dependencies for chart "empty":
🛳  - Installing chart "shared-dep-empty" as "my-shared-dep" in namespace "my-shared-dep-ns"…
🛳  Installing chart "empty" as "my-hypper-name" in namespace "hypper"…
👏 Done!
```

I think the clapping-hands emoji is 2 characters wide, and in some places it gets shown in one way, and in others, the other way. What a mess! :man_shrugging: 

I wonder how the ouput from this PR looks in other terminals besides mine! here in xfce and gnome term, it looks ok with the PR change xD.

Reply:
:shrug: not a blogger for me, just wanted to point that out, I guess output largely depends on the used terminal....